samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/Program.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using System.IO;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Data.DataView;
+using AnomalyDetection.DataStructures;
+using System.Linq;
+using static Microsoft.ML.Transforms.NormalizingEstimator;
+using System.IO.Compression;
+using System.Collections.Generic;
+
+namespace CreditRisk
+{
+    class Program
+    {
+        private static string BaseDatasetsRelativePath = @"../../../../Data";
+        private static string DataRelativePath = $"{BaseDatasetsRelativePath}/german.data.txt";
+
+        private static string DataPath = GetAbsolutePath(DataRelativePath);
+
+        private static string BaseModelsRelativePath = @"../../../../MLModels";
+        private static string ModelRelativePath = $"{BaseModelsRelativePath}/GermanCreditCardModel.zip";
+
+        private static string ModelPath = GetAbsolutePath(ModelRelativePath);
+
+        static void Main()
+        {
+            // 1. Implement the pipeline for creating and training the model    
+            MLContext mlContext = new MLContext(seed:0);
+
+            // 2. Specify how training data is going to be loaded into the DataView
+            IDataView dataView = mlContext.Data.LoadFromTextFile<CustomerCreditData>(DataPath, separatorChar: ' ');
+            //var peak1 = dataView.Preview();
+
+            var trainTestData = mlContext.AnomalyDetection.TrainTestSplit(dataView, 0.2);
+            var trainData = trainTestData.TrainSet;
+            var testData = trainTestData.TestSet;
+
+            //count the number of anomalies in test data
+            var t=mlContext.Data.CreateEnumerable<CustomerCreditData>(testData, reuseRowObject: false)
+                                           .Where(x => x.Label == 2)              
+                                           .ToList().Count;
+
+
+            //var peak2 = trainTestData.TrainSet.Preview();
+            //var peak3 = trainTestData.TestSet.Preview();
+            //string[] featureColumnNames = dataView.Schema.AsQueryable().Select(column => column.Name).ToArray();
+
+            // 2. Create a pipeline to prepare your data
+            // 2a. There are 2 types of features in the data. 1.Numerical and 2.Categorical.
+            //Transform Categorical features into Numeric values
+            //Concatenate numeric features and transformed categorical features
+            //Drop Label column from the pipeLine because Anomaly Detection trainer trains on unsupervised data.
+            IEstimator<ITransformer> dataProcessingPipeline = mlContext.Transforms.Concatenate("NumFeatures", nameof(CustomerCreditData.numOfMonths),
```

Comment:
NumFeatures means number of features for most people. Please use NumericFeatures instead.

Reply:
Changed